deploy/crds/alerts_v1alpha1_storagealert_cr.yaml

```diff
@@ -11,22 +11,22 @@ spec:
     serviceMonitor: "rook-ceph-mgr"
   prometheus:
     namespace: "openshift-monitoring"
-    label: "prometheus:k8s"
+    label: {"prometheus:k8s"}
```

Comment:
Since the type of the 'label' (in storagealert_types.go file) is `map[string]string`, you may have to change yaml to `label` line to
```yaml
label: {"prometheus": "k8s"}
```
If you multiple values, use `comma` to separate each key-value pair

Reply:
ack